libindy-crypto/src/ffi/bn.rs

```diff
@@ -0,0 +1,78 @@
+use bn::BigNumber;
+
+use ffi::ErrorCode;
+use errors::ToErrorCode;
+use std::slice;
+
+#[no_mangle]
+pub extern fn indy_crypto_primality_check(big_endian_number: *const u8,
```

Comment:
I want to ask for some changes to follow design and code structure. I believe we need to add is_prime method to BN entity instead of providing static non-entity related call. Now you provides different Rust API and FFI API.

It will allow to expose big nums in more clean and re-usable way. FFI API can look like this:

```
let bn = null;
ind_crypto_bn_new(bytes, &bn) 
let is_prime = indy_crypto_bn_is_prime(bn)
indy_crypto_free(bn)
```

it will later allow to expose complete big nums API

Reply:
The same need to be added to python wrapper. Just follow the same entity model as Rust API.

Reply:
Which `BN` entity?